packages/xo-server-sdn-controller/src/ovsdb-client.js

```diff
@@ -372,8 +371,7 @@ export class OvsdbClient {
       return false
     }
 
-    for (let i = 0; i < selectResult.options[1].length; ++i) {
-      const option = selectResult.options[1][i]
+    for (const option of selectResult.options[1]) {
```

Comment:
Can you do a separate PR that we'll merge first for all these unrelated changes?

Reply:
https://github.com/vatesfr/xen-orchestra/pull/4334